pkg/pipeline/conntrack/conntrack.go

```diff
@@ -141,14 +142,20 @@ func (ct *conntrackImpl) Track(flowLogs []config.GenericMap) []config.GenericMap
 			ct.connStore.addConnection(computedHash.hashTotal, conn)
 			ct.updateConnection(conn, fl, computedHash)
 			if ct.shouldOutputNewConnection {
-				outputRecords = append(outputRecords, conn.toGenericMap())
+				record := conn.toGenericMap()
+				addHashField(record, computedHash.hashTotal)
+				addTypeField(record, api.ConnTrackOutputRecordTypeName("NewConnection"))
+				outputRecords = append(outputRecords, record)
 			}
 		} else {
 			ct.updateConnection(conn, fl, computedHash)
 		}
 
 		if ct.shouldOutputFlowLogs {
-			outputRecords = append(outputRecords, fl)
+			record := fl.Copy()
+			addHashField(record, computedHash.hashTotal)
+			addTypeField(record, api.ConnTrackOutputRecordTypeName("FlowLog"))
```

Comment:
Should the values of ConnTrackOutputRecordTypeName be made into an enum and included in the documentation?